File: Learnum.ERP.Shared/Entities/Models/WorksheetDetailsModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Learnum.ERP.Shared.Entities.Models
{
    public class WorksheetDetailsModel: BaseModel
    {
        public long? WorkId { get; set; }
        public string Name { get; set; }
        public string Email { get; set; }
        public string Date { get; set; }
        public string Role { get; set; }
        public bool? IsActive { get; set; }

    }
}

File: Learnum_ERP UI/src/app/tds/component/hrd/daily-work/add-worksheet/worksheetdetails.model.ts
export class WorksheetDetailsModel {
    workId: number | null;
    name: string;
    email: string;
    date: string;
    role: string;
    TodaysWork :string;
    isActive: boolean | null;
    addedBy: number | null;
    addedDate: Date | null;
    updatedBy: number | null;
    updatedDate: Date | null;
    computerName: string;
}